### PR TITLE
onsetfix

### PR DESCRIPTION
**Pull request by xpander53**

### CODE REVIEW

File: src/main/resources/hl7/resource/Condition.yml [hunk 1]
```diff
@@ -154,12 +154,12 @@ onsetDateTime:
   expressionType: HL7Spec
   
 onsetString:
-  condition: $onsetDateTime NOT_NULL
+  condition: $onsetDateTime NULL
   type: STRING
   valueOf: PRB.17
```
Comment (klwhaley) [inline src/main/resources/hl7/resource/Condition.yml:159]: concerned about value of... being only PRB.17...shoudln't there be a DG1 value if we use DG1.5?

Comment (klwhaley) [reply src/main/resources/hl7/resource/Condition.yml:159]: After discussing, remove DG1.5 and add comment:
# DG1.5 not referenced for onsetDateTime as there is currently not a fallback DG1 segment to use for a onset string

**Review by klwhaley — APPROVED** [review]
Approving with comment

File: src/main/resources/hl7/resource/Condition.yml (hunk 1 above)
Comment (xpander53) [reply src/main/resources/hl7/resource/Condition.yml:159]: Removing DG1.5 and added a comment.

**Pull request merged**